test/labeling/model/test_label_model.py

```diff
@@ -242,45 +244,97 @@ def test_model_loss(self):
         L = np.array([[0, -1, 0], [0, 1, 0]])
         label_model = LabelModel(cardinality=2, verbose=False)
 
-        label_model.train_model(L, n_epochs=1, lr=0.01, momentum=0.9)
+        label_model.fit(L, n_epochs=1)
         init_loss = label_model._loss_mu().item()
 
-        label_model.train_model(L, n_epochs=10, lr=0.01, momentum=0.9)
+        label_model.fit(L, n_epochs=10)
         next_loss = label_model._loss_mu().item()
 
         self.assertLessEqual(next_loss, init_loss)
 
         with self.assertRaisesRegex(Exception, "Loss is NaN."):
-            label_model.train_model(L, n_epochs=10, lr=1e8)
+            label_model.fit(L, n_epochs=10, optimizer_config={"lr": 1e8})
```

Comment:
we can initialize LabelModel again here so the lr doesn't have to passed in like this (we don't want to encourage this usage right?)